### PR TITLE
refactor(support): move support tickets to manager-models

### PR DESCRIPTION
**Pull request by frenautvh**
extract ticket & ticket.constant from manager-support to manager-models (in order to reuse it in manager-hub)

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report installation#9809.0 ✘  
* Stage 1
  * Install ✘

File: packages/manager/modules/support/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   },
   "peerDependencies": {
     "@ovh-ux/manager-core": "^7.6.2",
+    "@ovh-ux/manager-models": "^0.0.0 || ^1.0.0",
```
Comment (marie-j) [inline packages/manager/modules/support/package.json]: I'm not sure it is needed as a peer in fact, it's not an `angularjs` module

Comment (frenautvh) [reply packages/manager/modules/support/package.json]: damn i'm so used to work with angularjs modules that i forgot it wasn't one!

File: packages/manager/modules/models/support/ticket/ticket.class.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+import { STATUS } from './ticket.constants';
 
-export default class SupportTicket {
+export default class {
   constructor(ticket) {
     Object.assign(this, ticket);
   }
 
+  static get Status() {
+    return { ...STATUS };
```
Comment (FredericEspiau) [inline packages/manager/modules/models/support/ticket/ticket.class.js]: I don't get what this is supposed to do

Comment (frenautvh) [reply packages/manager/modules/models/support/ticket/ticket.class.js]: maybe not, but some people like having a lots of files ^^

**Pull request merged**